docs/rfc/README.md

```diff
@@ -0,0 +1,34 @@
+# Requests for Comments
```

Comment:
Don't we need the following at the beginning of this file?

```
---
sidebar_position: 0
---
```

Reply:
adding thanks